core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -134,6 +140,10 @@ public JDBCUserStoreManager(RealmConfiguration realmConfig, int tenantId) throws
 //		}
 
         // new properties after carbon core 4.0.7 release.
+        if (hashProvider == null) {
+            hashProvider = initializeHashProvider(realmConfig.getUserStoreProperties());
+        }
+
```

Comment:
These lines should go before the comment right? otherwise the comment will not make any sense

Reply:
resolved